src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -169,4 +173,18 @@ public VodqaScreen scrollVerticallyByPercentage(int fromPercentHeight, int toPer
         visually.checkWindow(SCREEN_NAME, "Screen scrolled down");
         return this;
     }
+
+    @Override
+    public VodqaScreen doubleTapOnElement() {
+        LOGGER.info("Performing Double tap on element in Double tap screen");
+        driver.waitTillElementIsVisible(byDoubleTapScreenXpath).click();
+        driver.doubleTap(driver.waitTillElementIsVisible(byDoubleTapElementXpath));
+        return this;
+    }
+
+    @Override
+    public boolean isDoubleTapSuccessful() {
+        LOGGER.info("Checking if double tap on element is successful");
+        return driver.isElementPresent(byDoubleTapSuccessfulXpath);
```

Comment:
add visual validations before and after the double tap action

Reply:
Added visual validations